tensorflow/lite/delegates/nnapi/nnapi_delegate.cc

```diff
@@ -655,7 +655,23 @@ NNMemory::NNMemory(const NnApi* nnapi, const char* name, size_t size) {
   if (name && size > 0) {
     nnapi_ = nnapi;
     byte_size_ = size;
+#ifdef __ANDROID__
     fd_ = nnapi_->ASharedMemory_create(name, size);
+#else
+    // Each call to ASharedMemory_create produces a unique memory space, hence
+    // name should not be used to create the shared memory file, otherwise
+    // two calls to create memory regions using the same 'name', will collide.
```

Comment:
maybe just simplify these comments to something like "Find a unique file name"?

Reply:
Agree, what about this one: 
    // For non-Android platforms ASharedMemory_create needs unique name to
    // create a shared memory object (see nnapi_implementation.cc).